### ISSUE TITLE
rdmc - not found in I3_PORTS when SYSTEM_PACKAGE=ON (Trac #928)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#922]
Migrated from https://code.icecube.wisc.edu/ticket/928
```json
{
    "status": "closed", 
    "changetime": "2019-01-11T23:52:58", 
    "description": "", 
    "reporter": "nega", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1547250778860196", 
    "component": "cmake", 
    "summary": "rdmc - not found in I3_PORTS when SYSTEM_PACKAGE=ON", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2015-04-14T19:45:16", 
    "milestone": "", 
    "owner": "nega", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150416T23:42:56: 
* nega commented:

> In changeset 2131:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2131"
> fall back to I3_PORTS if not found with SYSTEM_PACKAGES
> 
> closes #922, #923, #925, #926
> ```


* nega changed resolution from "" to "fixed"
* nega changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190111T23:52:58: nwhitehorn commented:

> In changeset 170039:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="170039"
> fall back to I3_PORTS if not found with SYSTEM_PACKAGES
> 
> closes #922, #923, #925, #926
> ```
